ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/type/SimpleTypeMapper.java

```diff
@@ -105,6 +106,10 @@ public static Schema getSchema(TypeDescKind typeKind) {
                 return schema;
             case ANYDATA:
                 return new Schema();
+            case NEVER:
+                // TODO: Check whether this can be represented with `not` in object schema
+                // Current behaviour will be skipping the type mapping
+                return null;
             default:
                 return null;
```

Comment:
```suggestion
            case NEVER:
            default:
                // TODO: For `NEVER`, check whether this can be represented with `not` in object schema
                // Current behaviour will be skipping the type mapping
                return null;
```